examples/stable-diffusion/text_to_image_generation.py

```diff
@@ -53,6 +55,13 @@ def main():
         help="Path to pre-trained model",
     )
 
+    parser.add_argument(
+        "--controlnet_model_name_or_path",
+        default="lllyasviel/sd-controlnet-canny",
+        type=str,
+        help="Path to pre-trained model",
+    )
```

Comment:
I would remove this and just keep `model_name_or_path` for specifying the model.
And you can override the default value as we do here for LDM3D: https://github.com/huggingface/optimum-habana/blob/04de0f59c3686d481b0f67d091768fa8b01f2851/examples/stable-diffusion/text_to_image_generation.py#L202

Reply:
Hi, We need two models for control net. The first model is stable diffusion model. The second model is the control net model. I have added examples of how to use 2 different control net models in the README both with stable diffusion 1.5.

Reply:
Added another example in README for SD2

Reply:
You're fully right, my bad